src/systems/player.rs

```diff
@@ -153,7 +153,7 @@ impl PlayerState {
                             )),
                         );
                         self.current = None;
-                        crate::write();
+                        crate::write().unwrap();
```

Comment:
Can you also avoid unwrapping, logging the error is better than crashing the app. Or at least print a good error message that explains the error

Reply:
sure thing, can do.